zvariant/src/type.rs

```diff
@@ -313,6 +314,64 @@ macro_rules! map_impl {
 map_impl!(BTreeMap<K: Ord, V>);
 map_impl!(HashMap<K: Eq + Hash, V, H: BuildHasher>);
 
+impl Type for Duration {
+    fn signature() -> Signature<'static> {
+        <(u64, u32)>::signature()
+    }
+}
+
+use std::num::{
+    NonZeroI16, NonZeroI32, NonZeroI64, NonZeroI8, NonZeroU16, NonZeroU32, NonZeroU64, NonZeroU8,
+};
```

Comment:
Either use the fully-qualified names (i-e remove this) or put this on the top with other imports.